app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
```

Comment:
Yeah. AFAICT, ng-show is the typical way of doing conditionals - if the condition in that attribute is true, the element will be shown. That's how I'm doing showing/hiding of the links. Easily moved to the entire Mapping element.